mne/io/tests/test_meas_info.py

```diff
@@ -279,6 +279,11 @@ def test_merge_info():
     # Check that you must supply Info
     assert_raises(ValueError, _force_update_info, info_a,
                   dict([('sfreq', 1000.)]))
+    # KIT System-ID
+    info_a['kit_system_id'] = 50
```

Comment:
> Actually, should it also fail if the other info does not have an id set (like it is here)? I'll leave that for you to decide.

This should not fail, this is our use case of merging EEG data and MEG data (MEG data would have the KIT-ID and EEG data would not)